openassessment/workflow/models.py

```diff
@@ -557,6 +564,8 @@ def update(self, submission_uuid, assessment_requirements):
             step_changed = True
 
         # Has the step received a score?
+        # If staff assessment is optional we will mark assessment as complete immediately.
+        # But if staff comes and assesses later, this date will not be updated.
```

Comment:
Is this okay or should the `assessent_completed_at` date be updated if staff gives an assessment after the state is `done`?